docs/hello_world.rst

```diff
@@ -1,246 +0,0 @@
-The purpose of this document is to showcase creation of a basic Gumby experiment.
```

Comment:
Why was my beautiful documentation removed? 😢

Reply:
Because it is based on Dispersy. I am planning on rewriting it and I should probably do that tomorrow.

Reply:
98% of this file is about general Gumby configuration, throwing it all out seems like a waste.

Reply:
You're right, I didn't read through it properly. I made a few changes to make the last section use IPv8 instead.